app/react/components/PageHeader/HeaderTitle.test.tsx

```diff
@@ -23,14 +25,17 @@ test('should not render without a wrapping HeaderContainer', async () => {
 test('should display a HeaderTitle', async () => {
   const username = 'username';
   const user = new UserViewModel({ Username: username });
+  const queryClient = new QueryClient();
 
   const title = 'title';
   const { queryByText } = render(
-    <UserContext.Provider value={{ user }}>
-      <HeaderContainer>
-        <HeaderTitle title={title} />
-      </HeaderContainer>
-    </UserContext.Provider>
+    <QueryClientProvider client={queryClient}>
```

Comment:
EE seems to use renderWithQueryClient - could you use that here so that they're in sync?